examples/tree_2d_dgsem/elixir_kpp.jl

```diff
@@ -38,15 +39,15 @@ end
 end
 
 # Wavespeeds
-@inline wavespeed(::KPPEquation2D) = 1.0
+@inline wavespeed(::KPPEquation2D) = 1.0 # mark
```

Comment:
Is that okay to make it fixed to `Float64` here?

Reply:
I think it's okay for this example. You could add a comment like

```suggestion
# Please note that the return type should be modified if other
# floating point types should be used.
@inline wavespeed(::KPPEquation2D) = 1.0
```